### PR TITLE
chore(test): use jest html snapshots

### PR DESCRIPTION
**Pull request by Haroenv**
meanwhile also update dependencies

cc @rayrutjes

### CODE REVIEW

File: __tests__/__snapshots__/sitemap.js.snap [hunk 1]
```diff
@@ -1,92 +1,152 @@
...
+
+<?xml version="1.0"
+      encoding="utf-8"
+?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url xmlns:xhtml="http://www.w3.org/1999/xhtml">
+    <loc>
+      https://www.example.com
+    </loc>
+    <xhtml:link rel="alternate"
```
Comment (Haroenv) [inline __tests__/__snapshots__/sitemap.js.snap]: @rayrutjes, this seems wrong...

**Comment by vvo** [comment]
Can you precise again why the default serializer from Jest is not good enough?

**Comment by Haroenv** [comment]
I would like to easily see diffs, and what I had earlier was confusing because of all the escaped `"` characters. I’m not sure if it’s a big deal, but I had this open on a branch on my computer and rather had it online too, to see what the effect is.

**Comment by vvo** [comment]
Since I did not experience the issue I am failing at understanding exactly the problem. But from the diff I see what you mean. So it's not really to be able to render HTML, it's more about avoiding characters escape from the default jest formatting.

**Comment by rayrutjes** [comment]
@vvo in addition to making sure there are no weird escaping strings, the formatter also outputs every directive / attribute / text node on a new line. This makes diffs even more easy to read.

File: __tests__/__snapshots__/sitemap.js.snap (hunk 1 above)
Comment (Haroenv) [inline __tests__/__snapshots__/sitemap.js.snap]: fixed now 😄

**Comment by vvo** [comment]
We are good here I guess

**Pull request merged**